### PR TITLE
Videos list, unseen videos counter badge and popup re-design

### PR DESCRIPTION
**Pull request by Betree**
# Features

## Unseen videos counter badge

* Reset when opening the popup
* Refreshed on cache update (when visiting YouTube)
* Can be disabled in settings

Preview

## Global re-design

* Use CaptainFact header image
* Use tabs to separate `Videos list` and `Settings`

MainPopupPreview

Preview

## Videos list

* The 4 most recent videos are fetched when opening the popup

Preview

## Settings

* Added a setting to disable unseen videos counter badge
* Re-design settings

Preview

### CODE REVIEW

File: app/components/VideosList.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+          if (data.allVideos.length === 0) {
+            return this.renderNoVideo()
+          }
+
+          return (
+            <div className={styles.videosList}>
+              {data.allVideos.map(({title, hashId, provider, providerId}) => (
+                <div key={hashId} className={styles.videoCard}>
+                  <ExternalLink href={linkToVerificationsPage(hashId)}>
+                    <img src={this.videoThumb(provider, providerId)} alt=""/>
```
Comment (Manu1400) [inline app/components/VideosList.js]: suggest : `provider` -> `provider_name`

File: app/lib/browser_icon_badge_counter.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+ * If value is 0, an empty string will be returned to hide the badge.
+ * If value is above `MAX_VALUE`, the string `${MAX_VALUE}+` is returned.
+ *
+ * @param {integer} valueInt
+ * @returns {string} the resulting string ready to be used in badge
+ */
+function encodeValue(valueInt) {
+  if (valueInt === 0) {
+    return ''
+  } else if (valueInt >= MAX_VALUE) {
```
Comment (Manu1400) [inline app/lib/browser_icon_badge_counter.js]: `99 >= 99` -> `true`, please change to `valueInt > MAX_VALUE` (for the unique case of value 99)

Comment (Betree) [reply app/lib/browser_icon_badge_counter.js]: This has been made on purpose, `99+` means `99 or more`.

File: chrome/manifest.dev.json [hunk 3]
```diff
@@ -41,5 +41,5 @@
     "*://m.youtube.com/*", 
     "*://www.youtube.com/*"
   ],
-  "content_security_policy": "default-src 'self'; script-src 'self' http://localhost:3000 https://localhost:3000 'unsafe-eval'; connect-src http://localhost:3000 https://localhost:3000 https://localhost:4003; style-src * 'unsafe-inline' 'self' blob:; img-src 'self' data:;"
+  "content_security_policy": "default-src 'self'; script-src 'self' http://localhost:3000 https://localhost:3000 'unsafe-eval'; connect-src http://localhost:3000 https://localhost:3000 https://localhost:4003; style-src * 'unsafe-inline' 'self' blob:; img-src 'self' https://img.youtube.com data:;"
```
Comment (Manu1400) [inline chrome/manifest.dev.json]: - `https://img.youtube.com` permission in manifest.dev.json : OK
- `https://img.youtube.com` permission in manifest.prod.json : FAIL (please add the change on manifest.prod.json)
- `https://img.youtube.com` permission in manifest.staging.json : FAIL (please add the change on manifest.staging.json)

Comment (Betree) [reply chrome/manifest.dev.json]: Well done, thanks!

File: app/components/VideosList.js (hunk 1 above)
Comment (Betree) [reply app/components/VideosList.js]: `provider` is used in REST + GraphQL APIs and in most of the code.

**Pull request merged**